src/bin/leftwm-check.rs

```diff
@@ -48,9 +47,6 @@ async fn main() -> Result<()> {
     match load_from_file(config_file, verbose) {
         Ok(config) => {
             println!("\x1b[0;92m    -> Configuration loaded OK \x1b[0m");
-            if config == Config::default() {
-                println!("\x1b[1;93mWARN: The file loaded was the default. Your configuration is likely invalid \x1b[0m");
```

Comment:
I'm not sure I follow this, what're we replacing this warning with? (Apologies if it's covered elsewhere).

Reply:
In the worker this code's logic did make sense because if the file failed to load, the default config will be loaded. So to detect that the config has been loaded properly, we would check that the config loaded is different than the default config.

This logic doesn't make sense here in leftwm-check because the `load_from_file` function returns an error if the file could not be read or could not be parsed. So if the configuration of the user is the same than the default configuration it only means they are using the default configuration. It doesn't mean that the configuration was invalid in any way.

```rust
    if Path::new(&config_filename).exists() {
        let contents = fs::read_to_string(config_filename)?; // Error if file could not be read
        if verbose {
            dbg!(&contents);
        }
        Ok(toml::from_str(&contents)?) // Error if file could not be parsed
    } else {
        bail!("Configuration not found in path"); // Error if file does not exists
    }
```

(Actually... :thinking: we can remove the check that the file exists. This is already covered by `fs::read_to_string()` and should display something like "file not found")

Reply:
Done in ceb1a78

Reply:
Ah; thank you for the explanation! Hadn’t tracked with the difference. :)